assets/css/style.css

```diff
@@ -52,6 +52,7 @@ body {
 }
 
 .b {
+    cursor: pointer;    
```

Comment:
Change cursor style so it is apparent that the breweries are clickable despite not having visible links.